frontend/src/components/NavBar/NavBar.js

```diff
@@ -77,6 +81,16 @@ const NavBar = () => {
     }
   };
 
+  useEffect(() => {
+    const handleResize = () => {
+        setIsMobile(mobile.matches);
+    };
+    mobile.addEventListener('change', handleResize);
+    return () => {
+        mobile.removeEventListener('change', handleResize);
+    };
```

Comment:
```suggestion
    const handleResize = () => {
      setIsMobile(mobile.matches);
    };
    mobile.addEventListener('change', handleResize);
    return () => {
      mobile.removeEventListener('change', handleResize);
    };
```
들여쓰기가 네 칸인데 prettier 설정이 잘 되어있으실까요?

Reply:
아이고.. 수정하였습니다! 또한 prettier 이용해 탭 사이즈를 2로 바꿀 수 있었습니다! 감사합니다!